cdci_osa_plugin/osa_dataserve_dispatcher.py

```diff
@@ -657,14 +657,20 @@ class OsaQuery(ProductQuery):
     
     def check_query_roles(self, provided_roles: List[str], par_dic: dict):
         # TODO: there is this get_par_by_name there, to check if it can be made work
-        
+        # the _parameters_list for the query should be set
         max_pointings = int(par_dic.get('max_pointings', 50))
+        # max_pointings = self.get_par_by_name('max_pointings').value
```

Comment:
what about this get_par_by_name, does it work?

Reply:
Not really, because `get_par_by_name` searches within the list of parameters for that specific query. And for this query those parameters are not set in the isgri instrument factory. 

That is why I was wondering if we should do that

Reply:
why not set?

Reply:
When calling:

[https://github.com/oda-hub/dispatcher-plugin-integral/blob/4608c9841124f213a1d26c279c8e15732d7d1cf0/cdci_osa_plugin/osa_isgri.py#L55](https://github.com/oda-hub/dispatcher-plugin-integral/blob/4608c9841124f213a1d26c279c8e15732d7d1cf0/cdci_osa_plugin/osa_isgri.py#L55)

that list of parameters is set only for the `instr_query`:

[https://github.com/oda-hub/dispatcher-plugin-integral/blob/4608c9841124f213a1d26c279c8e15732d7d1cf0/cdci_osa_plugin/osa_isgri.py#L63-L69](https://github.com/oda-hub/dispatcher-plugin-integral/blob/4608c9841124f213a1d26c279c8e15732d7d1cf0/cdci_osa_plugin/osa_isgri.py#L63-L69)

but not for the rest of product queries:

[https://github.com/oda-hub/dispatcher-plugin-integral/blob/4608c9841124f213a1d26c279c8e15732d7d1cf0/cdci_osa_plugin/osa_isgri.py#L94](https://github.com/oda-hub/dispatcher-plugin-integral/blob/4608c9841124f213a1d26c279c8e15732d7d1cf0/cdci_osa_plugin/osa_isgri.py#L94)

So, I was wondering if we could provide those parameters also for the product queries, or they should be set only for the instrument query.

Reply:
Maybe. Ok well I'd not bother with this now.